### PR TITLE
Word widths on justified line of text were calculated incorrectly.

### PR DESCRIPTION
**Pull request by belfz**
The width of words was incorrectly calculated for justified line of text. Please see the comments to the code changes I committed.

### CODE REVIEW

File: src/shapes/text.class.js [hunk 1]
```diff
@@ -500,7 +500,7 @@
       // stretch the line
       var words = line.split(/\s+/),
           charOffset = 0,
-          wordsWidth = this._getWidthOfWords(ctx, words.join(''), lineIndex, 0),
+          wordsWidth = this._getWidthOfWords(ctx, words.join(' '), lineIndex, 0),
```
Comment (belfz) [inline src/shapes/text.class.js]: Hence, the solution was to preserve the space characters.

**Comment by asturur** [comment]
can you correct the lint error?

I m going to try this code, it looks good.
I have been very busy and i m behind with issue tracker one week.
As soon as i catch up i merge this i think.

itext needs a full rewrite in my opinion.

**Comment by belfz** [comment]
Thank you @asturur! I was about to fix the lint error this morning, but I saw that you already did that and merged the fix. Appreciate that!

**Comment by asturur** [comment]
i wanted to ship in 1.6.7
 i guess 1.7.0 will take longer or will be bugged. so better to have as much as possible in this release

**Pull request merged**